yarn-project/noir-contracts/src/contracts/card_game_contract/src/game.nr

```diff
@@ -31,36 +31,25 @@ struct Game {
 global GAME_SERIALIZED_LEN: Field = 15;
 
 fn deserializeGame(fields: [Field; GAME_SERIALIZED_LEN]) -> Game {
-    let players = [
-        PlayerEntry {
-            address: fields[0],
-            deck_strength: fields[1] as u32,
-            points: fields[2] as u120,
-        },
-        PlayerEntry {
-            address: fields[3],
-            deck_strength: fields[4] as u32,
-            points: fields[5] as u120,
-        },
-    ];
-    let rounds_cards = [
-        Card::from_field(fields[6]), Card::from_field(fields[7]),
-        Card::from_field(fields[8]), Card::from_field(fields[9]),
-    ];
-    Game { 
+    let players = [PlayerEntry { address: fields[0], deck_strength: fields[1] as u32, points: fields[2] as u120 },
+        PlayerEntry { address: fields[3], deck_strength: fields[4] as u32, points: fields[5] as u120 }];
+    let rounds_cards = [Card::from_field(fields[6]),
+        Card::from_field(fields[7]),
+        Card::from_field(fields[8]),
+        Card::from_field(fields[9])];
+    Game {
         players,
         rounds_cards,
         started: fields[10] as bool,
         finished: fields[11] as bool,
         claimed: fields[12] as bool,
         current_player: fields[13] as u32,
-        current_round: fields[14] as u32,
+        current_round: fields[14] as u32
     }
 }
 
 fn serializeGame(game: Game) -> [Field; GAME_SERIALIZED_LEN] {
-    [
-        game.players[0].address,
+    [game.players[0].address,
```

Comment:
damn the formatter doesn't format well with arrays. Wish the next lines were in the same level as this!

Reply:
This PR uses an old version, master formats like this: https://github.com/f01dab1e/aztec-packages/blob/1f6d194c2ed25cdafe93bb1af684aa05f05596b5/yarn-project/noir-contracts/src/contracts/card_game_contract/src/game.nr#L53-L71

Reply:
I confirm that new nargo didn't do this change.